alfalfa_worker/jobs/openstudio/create_run.py

```diff
@@ -46,6 +46,13 @@ def exec(self):
         # run workflow
         check_output(['openstudio', '--gem_path', '/var/lib/gems/2.7.0', 'run', '-m', '-w', str(submitted_osw_path)])
 
+        self.logger.info('Loading Building Metadata')
```

Comment:
Runs now contain the name of the run. This is extracted by a measure which creates a report including this and possibly other data in the future.